erc20/alias-erc20-deploy.js

```diff
@@ -54,48 +54,48 @@ async function deploy(info) {
-  await deploy(conf.child);
-  await deploy(conf.parent);
+  await deploy(conf.bridges[0].child.url, conf.sender.child, conf.contract.child);
+  await deploy(conf.bridges[0].parent.url, conf.sender.parent, conf.contract.parent);
 
   // add minter
-  await conf.child.newInstance.methods.addMinter(conf.child.bridge).send({ from: conf.child.sender, to: conf.child.bridge, gas: 100000000, value: 0 });
-  await conf.parent.newInstance.methods.addMinter(conf.parent.bridge).send({ from: conf.parent.sender, to: conf.child.bridge, gas: 100000000, value: 0 });
-
-  // register operator
-  await conf.child.newInstanceBridge.methods.registerOperator(conf.child.operator).send({ from: conf.child.sender, gas: 100000000, value: 0 });
-  await conf.parent.newInstanceBridge.methods.registerOperator(conf.parent.operator).send({ from: conf.parent.sender, gas: 100000000, value: 0 });
+  await conf.contract.child.newInstance.methods.addMinter(conf.contract.child.bridge).send({ from: conf.sender.child.address, to: conf.contract.child.bridge, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstance.methods.addMinter(conf.contract.parent.bridge).send({ from: conf.sender.parent.address, to: conf.contract.parent.bridge, gas: 100000000, value: 0 });
 
   // register token
-  await conf.child.newInstanceBridge.methods.registerToken(conf.child.token, conf.parent.token).send({ from: conf.child.sender, gas: 100000000, value: 0 });
-  await conf.parent.newInstanceBridge.methods.registerToken(conf.parent.token, conf.child.token).send({ from: conf.parent.sender, gas: 100000000, value: 0 });
+  await conf.contract.child.newInstanceBridge.methods.registerToken(conf.contract.child.token, conf.contract.parent.token).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstanceBridge.methods.registerToken(conf.contract.parent.token, conf.contract.child.token).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
+
+  for (const bridge of conf.bridges) {
+    // register operator
+    await conf.child.newInstanceBridge.methods.registerOperator(bridge.child.operator).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+    await conf.parent.newInstanceBridge.methods.registerOperator(bridge.parent.operator).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
+
+    const alias = "MYBRIDGE2"
+    const url = bridge.child.url;
+    log = 'registering bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_registerBridgeByAlias', [alias, conf.contract.child.bridge, conf.contract.parent.bridge]);
+
+    log = 'subscribing bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_subscribeBridgeByAlias', [alias]);
+
+    log = 'register token to subbridge..'
+    await jsonRpcReq(url, log, 'subbridge_registerTokenByAlias', [alias, conf.contract.child.token, conf.contract.parent.token]);
+  }
+
+  // setOperatorThreshold
```

Comment:
// set operator threshold

Reply:
This PR will be closed and this review is resolved in the followed PR #8. Thus marking this review resolved.